spec/helpers/blacklight_helper_spec.rb

```diff
@@ -128,4 +132,32 @@
       expect(helper).to have_received(:head).with(:bad_request)
     end
   end
+
+  describe "#link_back_to_catalog_safe" do
+    let(:blacklight_config) { CatalogController.new.blacklight_config }
```

Comment:
@hackartisan I am now using Blacklight built-in classes for `blacklight_config` and `search_state`, this should make the test easier to follow.